docs/eventing/channels/README.md

```diff
@@ -10,24 +10,29 @@ procedure:
     ```
 -->
 
-Channels are Kubernetes Custom Resources which define a single event forwarding
-and persistence layer. Messaging implementations may provide implementations of
-Channels via the
+Channels are Kubernetes Custom Resources which define a single event forwarding and persistence layer.
+Messaging implementations may provide implementations of Channels via the
 [ClusterChannelProvisioner](https://github.com/knative/eventing/blob/master/pkg/apis/eventing/v1alpha1/cluster_channel_provisioner_types.go#L35)
-object, supporting different technologies, such as Apache Kafka or NATS
-Streaming.
+object, supporting different technologies, such as Apache Kafka or NATS Streaming.
 
 This is a non-exhaustive list of Channels for Knative.
 
+
 ### Inclusion in this list is not an endorsement, nor does it imply any level of support.
 
+
 ## Channels
 
 These are the channels `CRD`s.
```

Comment:
I'm not sure I'm understanding why some of these are marked "CRD" and some are marked "CCP". Are they not all custom objects?

Reply:
The answer is because pre 0.7 we only supported Provisioner based, in 0.7 we introduced the CRD based ones. For ensuring continuity with our customers, we're supporting both models in 0.8 and hence both models need to be supported. In 0.9 we are removing support for Provisioner models.